driver/src/main/com/mongodb/client/MongoDatabase.java

```diff
@@ -191,4 +191,19 @@
      */
     void createCollection(String collectionName, CreateCollectionOptions createCollectionOptions);
 
+    /**
```

Comment:
Nice - that also means I can move all the code under `com.mongodb.client.gridfs`